src/assets/CollectiblesController.test.ts

```diff
@@ -1132,11 +1132,66 @@ describe('CollectiblesController', () => {
     });
 
     describe('checkAndUpdateCollectiblesOwnershipStatus', () => {
-      it('should check whether collectibles for the current selectedAddress/chainId combination are still owned by the selectedAddress and update the isCurrentlyOwned value to false when collectible is not still owned', async () => {
```

Comment:
The diff on this file got a bit screwed up again because I added a new layer of describes (one for each of `checkAndUpdateAllCollectiblesOwnershipStatus` and `checkAndUpdateSingleCollectibleOwnershipStatus`